tests/lib/rules/no-extra-parens.js

```diff
@@ -152,7 +154,6 @@ ruleTester.run("no-extra-parens", rule, {
         "var o = { foo: (function() { return bar(); })() };",
         "o.foo = (function(){ return bar(); })();",
         "(function(){ return bar(); })(), (function(){ return bar(); })()",
-        "((function(){ return bar(); })())",
```

Comment:
I think this test case is not a regression test.
This is added in https://github.com/eslint/eslint/commit/9ea75573c95bb3bd66f922bf19bab500962162eb.
The commit merges `no-extra-parens` and `no-wrap-func`.
However, `no-wrap-func` had not been having this test case: https://github.com/eslint/eslint/blob/v0.24.1/tests/lib/rules/no-wrap-func.js